components/org.wso2.carbon.stream.processor.core/src/main/resources/queries.yaml

```diff
@@ -61,7 +61,7 @@ queries:
 
   -
    mappings:
-      CREATE_TABLE: CREATE TABLE {{TABLE_NAME}} (id INT NOT NULL IDENTITY(1,1) PRIMARY KEY, siddhiAppName VARCHAR(100), revision VARCHAR(300), snapshot VARBINARY(8000))
+      CREATE_TABLE: CREATE TABLE {{TABLE_NAME}} (id INT NOT NULL IDENTITY(1,1) PRIMARY KEY, siddhiAppName VARCHAR(100), revision VARCHAR(300), snapshot VARBINARY(max))
```

Comment:
why changing this value ?

Reply:
To resolve buffer overflow error [#762](https://github.com/wso2/product-sp/issues/762)

Reply:
ack